client/src/components/Draftail/Draftail.scss

```diff
@@ -107,13 +108,32 @@ $draftail-editor-font-family: $font-sans;
 }
 
 .Draftail-Toolbar {
-  border: 1px solid $color-grey-3;
+  border: none;
+  // Remove once we drop support for Safari 14.
+  // stylelint-disable-next-line property-disallowed-list
+  border-bottom-left-radius: 0;
+  border-end-start-radius: 0;
+  // Remove once we drop support for Safari 14.
+  // stylelint-disable-next-line property-disallowed-list
+  border-bottom-right-radius: 0;
+  border-end-end-radius: 0;
+  background-color: $draftail-editor-background;
+  color: $draftail-placeholder-text;
+
+  .Draftail-Editor--focus & {
+    color: $draftail-editor-text;
+  }
 }
 
 .Draftail-MetaToolbar {
   position: absolute;
   inset-inline-end: 0;
   visibility: hidden;
+  background-color: transparent;
+
+  &:empty {
+    display: none;
+  }
```

Comment:
Unrelated bug fix – realised our focus outline gets cropped by this toolbar, even when the toolbar is empty.